### ISSUE TITLE
Make Webmail and Map more mobile-friendly 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#297]
MCLV
Making webmail and map more mobile-friendly. as in actually being able to enter username and password in webmail and selecting ground/moon/mars/space or entering text in the search field. Entering text seems impossible on any of the web interfaces on android/chrome

**Comment by SwissalpS** [comment]
I sympathise with the wish for webmail to work on small devices.
The map however, not so much. Some hardware tools are not much use for certain tasks. Yes you can unscrew a bigger screw with a smaller screwdriver, but does it make sense to do so?

**Comment by OgelGames** [comment]
How about the new web UI, that will replace webmail? https://pandorabox.io/ui/

As for the map, I partly agree with SwissalpS that it's not intended for mobile, but it still could be improved a bit, at least making it functional.

**Comment by MCLV-pandorabox** [comment]
I disagree with SwissalpS, and by extension OgelGames on the map issue.
If I understand correctly the map shouldn't be used on mobile, and has no practical use, therefore nothing needs to change?
I strongly disagree with that statement.
It should be up to the user to decide if it is practical to use. It is however up to the developer/team to make it available or not.
If the map was indeed not intended for the mobile user, then present them with a message and explanation and don't present them buttons that can't be pressed. (this is a general development guideline, is it not?)

Besides I see many uses for this on "mobile" because mobile is not only phone, it is also tablet and some people just don't have access to a desktop or laptop, or prefer not to.

In web development we currently employ a "mobile first" strategy. And I see from pandorabox.io/ui that pandorabox development adheres to this. Which also solves my issues with mail. 
Maybe users going to pandorabox.io/webmail should be redirected here instead?

* please note that any serious complaint from my part comes with the automatic offer to help improve the it

**Comment by OgelGames** [comment]
> If I understand correctly the map shouldn't be used on mobile, and has no practical use, therefore nothing needs to change?

Quite the opposite, it's definitely useful, and I think something should be done about it.

I don't know if @BuckarooBanzay has any plans for improving it, but he's the one that made it.

**Comment by BuckarooBanzay** [comment]
> If I understand correctly the map shouldn't be used on mobile, and has no practical use

Not really, i just developed it without any focus on mobile and some things may not work yet :wink: 

> In web development we currently employ a "mobile first" strategy. And I see from pandorabox.io/ui that pandorabox development adheres to this

Interesting, i only really tested it on the desktop browser :thinking: 

> I don't know if @BuckarooBanzay has any plans for improving it, but he's the one that made it.

Plans: sure, time to implement those properly: heh, not really

Feel free to open issues (or even PR's)

As for the current state of the webmail-app: The plan is to transition to the `mtui` app and extend that further (notifications, relay, etc)
I'll soon sunset the "old" webmail app anyway, too much nodejs stuff accumulated there :/

If you have new issues/requests, please direct them here: https://github.com/minetest-go/mtui
